java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -2648,16 +2608,34 @@ private static List<Arguments> getStreamAddOptions() {
     }
 
     @SneakyThrows
-    @ParameterizedTest
+    @ParameterizedTest(name = "{0}")
     @MethodSource("getStreamAddOptions")
-    public void xadd_with_options_returns_success(Pair<StreamAddOptions, String[]> optionAndArgs) {
+    public void xadd_with_options_to_arguments(
```

Comment:
excellent
optionally, we can move this to a separate file (we test options here, not client)

Reply:
I was thinking that too.  It this becomes a pattern, I will move the tests.